app/packages/components/src/components/Toast/Toast.tsx

```
@@ -29,27 +66,38 @@
   const action = (
     <div>
       <Box display="flex" justifyContent="flex-end">
-        {primary(setOpen)} {/* Pass setOpen to primary button */}
-        {secondary(setOpen)} {/* Pass setOpen to secondary button */}
+        {/* note: Not implemented within Python Panels context */}
+        {primary && primary(setOpen)} {/* Pass setOpen to primary button */}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)